libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt

```diff
@@ -375,6 +390,7 @@ class RouterWrapperTests {
             options
         ).routes().first()
 
+        println("[ddlog] expected: ${expected.toJson()}")
```

Comment:
This should be cleaned up.

Reply:
suppose we should automate this kind of checks(ktlint is good to use). Cut the ticket https://github.com/mapbox/navigation-sdks/issues/1932